examples/plotting/data_models.py

```diff
@@ -1,3 +1,10 @@
+'''This example shows creating sinusoidal wave using bokeh DataModel.
```

Comment:
nit: consistency with docstring format everywhere else in the repo
```suggestion
''' This example shows creating sinusoidal wave using bokeh DataModel.
```